src/wp-includes/class-wp-admin-bar.php

```diff
@@ -50,9 +50,9 @@ public function initialize() {
 			}
 		}
 
-		add_action( 'wp_head', 'wp_admin_bar_header' );
+		add_action( 'wp_head', 'wp_admin_bar_header', 7 );
 
-		add_action( 'admin_head', 'wp_admin_bar_header' );
+		add_action( 'admin_head', 'wp_admin_bar_header', 7 );
```

Comment:
This priority change will break some plugins: https://wpdirectory.net/search/01H9VJY3GR3M7QXRAYRE7RBW3T

Reply:
Not sure I see a way around this. Should I revert?

Reply:
Only way I can think to work around this is to leave it running at priority 10, but then also add it to run at priority 7. Then the function could be modified to add the following code:

```php
if ( 
    ! has_action( 'admin_head', 'wp_admin_bar_header', 10 ) || 
    ! has_action( 'admin_head', 'wp_admin_bar_header', 7 ) 
) {
    return;
}
remove_action( 'admin_head', 'wp_admin_bar_header', 10 );
```

This would ensure that if someone unhooked the function at the old location or the new location, that the it would short-circuit at either location. And then the `remove_action()` call will ensure that the function isn't run twice. Not ideal.